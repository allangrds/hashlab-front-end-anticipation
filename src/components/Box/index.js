import React from 'react'
import PropTypes from 'prop-types'

import styles from './styles.module.css'

const Box = ({ children }) => (
  <div className={styles.box}>
    { children }
  </div>
)

Box.propTypes = {
  children: PropTypes.node.isRequired,
}

export default Box
