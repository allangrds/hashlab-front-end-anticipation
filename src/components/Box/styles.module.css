@import "../../assets/css/settings/colors.css";
@import "../../assets/css/tools/fonts.css";

.box {
  background: var(--color-white);
  border: 1px solid var(--color-gray-3);
  box-sizing: border-box;
  border-radius: 4px;
}
