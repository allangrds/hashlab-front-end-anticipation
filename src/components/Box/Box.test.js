import React from 'react'
import { render } from '@testing-library/react'

import Box from '.'

test('Box should render and be updated correctly', () => {
  const text = 'Olá mundo'

  const {
    container,
    getByText,
  } = render(
    <Box>
      <p>Olá mundo</p>
    </Box>,
  )

  getByText(text)
  expect(container).toMatchSnapshot()
})
