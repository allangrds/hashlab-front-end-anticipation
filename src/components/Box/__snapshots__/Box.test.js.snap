// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Box should render and be updated correctly 1`] = `
<div>
  <div
    class="box"
  >
    <p>
      Ol√° mundo
    </p>
  </div>
</div>
`;
